src/ehttpc.erl

```diff
@@ -532,22 +552,49 @@ upgrade_requests(Map) when is_map(Map) ->
         pending => queue:new(),
         pending_count => 0,
         sent => Map,
-        prioritise_latest => false
+        prioritise_latest => false,
+        max_sent_expire => 0
```

Comment:
Oops, I forgot to remove `{up,down}grade_requests` when removing hot upgrade support.  I believe this is dead code now.  :see_no_evil:

Reply:
I will send another PR to clean up.

Reply:
https://github.com/emqx/ehttpc/pull/58